CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## 0.2.1 (2024-04-05)
```

Comment:
No need to bump version, I will do it on release. But thanks for CHANGELOG!

Reply:
Andrey, hey! Can you make a release right after the PR has been merged?